test-clients/src/main/java/com/hedera/services/bdd/suites/token/UniqueTokenManagementSpecs.java

```diff
@@ -145,48 +151,84 @@ private ByteString metadata(String contents) {
 		return ByteString.copyFromUtf8(contents);
 	}
 
+	private HapiApiSpec distinguishesFeeSubTypes() {
+		return defaultHapiSpec("happyPathFiveMintOneMetadata")
+				.given(
+						newKeyNamed(SUPPLY_KEY),
+						cryptoCreate(TOKEN_TREASURY),
+						cryptoCreate("customPayer"),
+						tokenCreate(NFT)
+								.tokenType(TokenType.NON_FUNGIBLE_UNIQUE)
+								.supplyType(TokenSupplyType.INFINITE)
+								.initialSupply(0)
+								.supplyKey("supplyKey")
+								.treasury(TOKEN_TREASURY),
+						tokenCreate(FUNGIBLE_TOKEN)
+								.tokenType(TokenType.FUNGIBLE_COMMON)
+								.supplyType(TokenSupplyType.FINITE)
+								.initialSupply(10)
+								.maxSupply(1100)
+								.supplyKey("supplyKey")
+								.treasury(TOKEN_TREASURY)
+				).when(
+						mintToken(NFT, List.of(ByteString.copyFromUtf8("memo"))).payingWith("customPayer").signedBy("customPayer", "supplyKey").via("mintNFT"),
+						mintToken(FUNGIBLE_TOKEN, 100L).payingWith("customPayer").signedBy("customPayer", "supplyKey").via("mintFungible")
+				).then(
+						UtilVerbs.withOpContext((spec, opLog) -> {
```

Comment:
Other than seeing that the transaction fees are different, can we use the `feeFor` method to verify additionally that the FeeData returned matches a given subType from the config and is it a meaningful check in your opinion?